client/src/js/pages/contactsPage.js

```diff
@@ -1,15 +1,11 @@
 const BasePage = require('watch-framework').BasePage;
+const StorageHub = require('watch-framework').StorageHub;
 const compiledTemplate = require('../../templates/contactsPage.hbs')
 
 class ContactsPage extends BasePage {
 
-  constructor(props = {}) {
-    super(props);
-    this.contacts = props.contacts || [
-      { name: 'Ray', phoneNumber: '0431 111 111' },
-      { name: 'Sinan', phoneNumber: '0431 222 222' },
-      { name: 'Jafari', phoneNumber: '0431 333 333' },
-    ];
```

Comment:
I don't think many students used the props to create new pages anyway, and is probably not something worth going into detail about.